sphinx/source/docs/user_guide/server.rst

```diff
@@ -1183,10 +1183,22 @@ distributed among individual servers.
     The Bokeh server is horizontally scalable. To add more capacity, you
     can run more servers behind a load balancer.
 
-Nginx can help with load balancing. This section describes some of the basics
-of one possible configuration, but please also refer to the
-`Nginx load balancer documentation`_. For instance, there are different
-strategies available for choosing what server to connect to next.
+You can run as many Bokeh servers as you need. In the following, let us
+suppose that you have three Bokeh servers running on separate ports:
```

Comment:
Following https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#principles-of-good-style, I'd replace the first person 'us' with something else:

```suggestion
You can run as many Bokeh servers as you need. The following examples
are based on a setup with three Bokeh servers running on three different
ports:
```